### PR TITLE
Sspbc Bid Adapter : add native asset mapping and floor updates

### PR DESCRIPTION
**Pull request by wojciech-bialy-wpm**
## Type of change
- [x] Bugfix
- [x] Feature

## Description of change
- improved asset mapping for native ads (using type and id)
- add wmin/hmin to image/icon assets
- allow adding more than one image asset
- append view trackers to all bids
- append clickTrackers to native bid
- do not append default jstracker (this is now done on the backend)
- fix: read floorprices for natives and video
- ensure that site id is passed as String
- update tests

### CODE REVIEW

**Review by musikele — COMMENTED** [review]
I'll wait for your answers before deciding if you want to merge as-is or if you want to improve the adapter

File: modules/sspBCBidAdapter.js
Comment (wojciech-bialy-wpm) [reply modules/sspBCBidAdapter.js:263]: Perhaps this was not show correctly in the merge? The comment pertains to getCurrency() method.
`const getCurrency = () => config.getConfig('currency.adServerCurrency') || DEFAULT_CURRENCY;`

**Comment by wojciech-bialy-wpm** [comment]
Let's merge as-is. I have a new version on adapter in-dev (move banner html creation to backend, fix tracking macros and a few other improvements) - I'll add ortb conversion to that as well.

File: modules/sspBCBidAdapter.js
Comment (musikele) [reply modules/sspBCBidAdapter.js:263]: you're right

**Review by musikele — APPROVED** [review]
LGTM

**Pull request merged**
